compose/ui/ui/build.gradle

```diff
@@ -209,6 +209,7 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
                 dependsOn(notMobileMain)
                 dependencies {
                     implementation(libs.kotlinStdlibJdk8)
+                    implementation(project(":collection:collection"))
```

Comment:
Let's just do it this way:

1. download the latest IDEA
2. check your use case with and without this line again (removing `out` and `.gradle` caches)
3. if this line indeed fixes your use case, let's keep it, but add a comment about the use case.

Because by the design it is not needed, and either the use case isn't valid, or we have a bug.

Reply:
We shouldn't add this dependency just to accommodate the way I run experiments. I'll check again whether normal code (e.g. in samples) works without it and remove if it does.

Reply:
Removed.